### PR TITLE
Add eslint

### PR DESCRIPTION
**Pull request by mariana-mendes**
#### What is the purpose of this pull request?
as the title says.

### CODE REVIEW

File: react/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
```
Comment (GustavoMA) [inline react/tsconfig.json]: This file is not necessary since it is a Javascript app :)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (luciannojunior) [inline package.json]: Remove it

Comment (GustavoMA) [inline package.json]: These configs in the root are not necessary, only need to add them in the `react/package.json`.

File: yarn.lock [hunk 3]
```diff
@@ -0,0 +1,1105 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (GustavoMA) [inline yarn.lock]: These configs in the root are not necessary, only need to add them in the `react/yarn.lock`.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- Add eslint configuration.
```
Comment (luciannojunior) [inline CHANGELOG.md]:
```suggestion
- `eslint` configuration. 
```

File: react/package.json [hunk 5]
```diff
@@ -15,7 +16,11 @@
   },
   "devDependencies": {
     "@vtex/test-tools": "^0.1.2",
-    "apollo-client": "^2.5.1"
+    "apollo-client": "^2.5.1",
+    "eslint": "^5.16.0",
+    "eslint-config-vtex-react": "^4.1.0",
+    "prettier": "^1.17.1",
+    "typescript": "^3.4.5"
```
Comment (jgfidelis) [inline react/package.json]: remove

Comment (GustavoMA) [reply react/package.json]: Actually, it is necessary. @lucasecdb knows better about it.

Comment (GustavoMA) [reply react/package.json]: If I'm not wrong it's a dependency to use eslint + prettier, even in JS projects.

File: react/package.json [hunk 6]
```diff
@@ -3,7 +3,8 @@
   "version": "0.0.0",
   "scripts": {
     "pretest": "yarn",
-    "test": "vtex-test-tools test"
+    "test": "vtex-test-tools test",
+    "lint": "eslint **.js"
```
Comment (jgfidelis) [inline react/package.json]: I tested and could not make this work, use:
`"lint": "eslint --ext js,jsx,ts,tsx .",`

File: react/package.json (hunk 5 above)
Comment (lucasecdb) [reply react/package.json]: yep, obligatory

**Pull request merged**